### PR TITLE
Added cladding Properties to TACOCAT

### PR DESCRIPTION
**Pull request by thurstonsd**
This addresses issue #62. This is an updated version to run clad_props.py on TACOCAT

### CODE REVIEW

File: TACOCAT-Run CladProps.py
Comment (lcarasik) [inline TACOCAT-Run CladProps.py:1]: This file needs to be remove, it was only used for testing.

**Comment by lcarasik** [comment]
There is a number of tacocat.py files with the full code, can someone figure out why there is 3 different ones instead of just tacocat.py?

**Comment by FranklinTrevor** [comment]
I went ahead and removed the extra TACOCAT files. Should be good now for your review @lcarasik

**Comment by FranklinTrevor** [comment]
All extra files are addressed and this should be good to merge

**Review by sierratutwiler — APPROVED** [review]
Looks good! The only thing I see is that you might need a comma on the last item in the cladding dictionary on line 66 in Clad_Props.py, I went ahead and added the comma and approved.

**Comment by lcarasik** [comment]
Hey folks,

Sorry can you update this one more time with the other pull request being completed?

**Pull request merged**